src/mantle/support/class-str.php

```diff
@@ -563,10 +599,20 @@ public static function substr_count( $haystack, $needle, $offset = 0, $length =
 	 * @param string $string The string to modify.
 	 * @return string
 	 */
-	public static function ucfirst( $string ) {
+	public static function ucfirst( string $string ) {
 		return static::upper( static::substr( $string, 0, 1 ) ) . static::substr( $string, 1 );
```

Comment:
No idea, deferring to laravel on this one

Reply:
So I actually looked this up after posting this, and it is because of multibytes. `ucfirst` isn't multibyte safe, so this is a well known workaround for that.